tests/test_schema_validation.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gordon.schema.validate import JsonValidator
+from gordon import exceptions
+
+import pytest
+
+
+@pytest.fixture
+def validator():
+    return JsonValidator()
+
+
+valid_audit_log = {
```

Comment:
Is there a reason to have these as constants, and not pytest fixtures? Do you think we could reuse them, i.e. making use of `conftest.py`? 

If we do keep these as constants, then the variable names should be all caps.

Reply:
They're not fixtures, because they are only used once in the annotation and fixtures can't be included in annotations. However, they're not really constants either, nor intended to be used as such (hence the no caps). The only reason they're declared as variables at all is to improve readability in the parametrize annotation, directly below.